app/src/main/java/com/pileproject/drive/setting/app/ProgramListFragment.java

```diff
@@ -80,17 +92,25 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         mProgramsCheckAllButton[0] = (Button) v.findViewById(R.id.programList_samplePrograms_checkAllButton);
         mProgramsCheckAllButton[1] = (Button) v.findViewById(R.id.programList_userPrograms_checkAllButton);
 
+
         return v;
     }
 
     @Override
     public void onActivityCreated(Bundle savedInstanceState) {
         super.onActivityCreated(savedInstanceState);
 
+        final ProgramListFragment thiz = this;
```

Comment:
DriveApplication.java can provide a context but it is not good way for this, I think. View v has its context and you can access it with getContext method.